### PR TITLE
[nats] Releases v2.10.3 and v2.9.23

### PR DESCRIPTION
**Pull request by wallyqs**
Details can be found here:

https://github.com/nats-io/nats-server/releases/tag/v2.10.3 
https://github.com/nats-io/nats-server/releases/tag/v2.9.23

### CODE REVIEW

**Comment by wallyqs** [comment]
@yosifkit would be great to get a merge on this one if all looks good, thanks!

**Comment by derekcollison** [comment]
This is for a Golang CVE that was released that could affect our users and customers. Thanks.

File: library/nats [hunk 1]
```diff
@@ -28,23 +28,23 @@ Architectures: windows-amd64
 Directory: 2.10.x/nanoserver-1809
```
Comment (yosifkit) [inline library/nats:28]: This `2.10*nanoserver-1809` entry should also be getting the `2.10.3` bump in its tags.

Comment (wallyqs) [reply library/nats:28]: Thanks @yosifkit, just updated.

**Comment by wallyqs** [comment]
Thanks @yosifkit !

**Pull request merged**